.github/workflows/frontend-verify.yml

```diff
@@ -105,16 +61,12 @@ jobs:
         with:
           go-version: '1.22'
           cache-dependency-path: backend/go.sum
-      - name: Setup Node
-        uses: actions/setup-node@v4
-        with:
-          node-version: '20'
       - name: Install frontend dependencies
         working-directory: frontend
-        run: npm ci
+        run: bun install
       - name: Install Playwright browsers
         working-directory: frontend
-        run: npm run install:browsers
+        run: bun run install:chromium
```

Comment:
We only need chrome, firefox/safari/webkit are taking up CI time but we don't care right now